xml/System.Runtime.InteropServices.JavaScript/JSMarshalerType.xml

```diff
@@ -763,7 +764,7 @@
         <ReturnType>System.Runtime.InteropServices.JavaScript.JSMarshalerType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Marshaling validates that the JavaScript function returned undefined value.</summary>
```

Comment:
```suggestion
        <summary>Gets a value that indicates whether the JavaScript function returned an undefined value.</summary>
```

Reply:
I'm not really understanding what this property does, so probably my suggestion needs to be reworded.